### DISCUSSION TITLE
cmd-shift or ctrl-shift to draw arrows

### DISCUSSION THREAD

**Discussion by mikeafish** [Softal-io/presentify-community#9] [Ideas]
If holding shift creates straight lines, another modifier key like cmd/ctrl or option/alt seems like a natural way to toggle an arrowhead

**Comment by rampatra** [comment]
Current key shortcut for Arrow is A, Free hand is F, Rectangle is R, and so on. 

However, if many users upvote this. I can think of adding this too.

**Reply by mikeafish** [reply]
Ah, I actually didn't know these existed since they're not in the customize shortcuts page of the settings

**Reply by rampatra** [reply]
You can press / or ? to see all Annotate related key shortcuts.

**Reply by rampatra** [reply]
I will also add a note in the Preferences regarding this. Thanks for bringing this up.

**Reply by rampatra** [reply]
I just checked the app and I have already added this in Preferences > Key Shortcuts. See below:
![CleanShot 2023-06-29 at 11 14 31@2x](https://github.com/rampatra/presentify-community/assets/2862724/dc10626d-7f2f-4349-8aad-aaad2466557c)
